### PR TITLE
Metal material overhaul

### PR DESCRIPTION
**Pull request by SirFancyBacon**
Overhauls metallic objects to make them look a bit more like metal!

**Features:**
- New normal map to help the light scattering effect.
- Reworked specularity and gloss for metallic textures.
- Reworked UVs for metallic objects.
- Add additional metallic objects.
- Renames metallic_2 to be correctly named metallic_1_light

**Screens:**
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by SirFancyBacon** [comment]
I am sure there will be more to come as i discover other metallic objects; But this should be ready for testing if nothing else

File: src/main/resources/rs117/hd/scene/model_overrides.json [hunk 1]
```diff
@@ -2759,7 +2759,7 @@
   },
   {
     "description": "Wintertodt Brazier",
-    "baseMaterial": "METALLIC_2_HIGHGLOSS",
+    "baseMaterial": "_HIGHGLOSS",
```
Comment (aHooder) [inline src/main/resources/rs117/hd/scene/model_overrides.json:2754]: Slight mistake here. Was this meant to be `METALLIC_2_HIGHGLOSS`?

Comment (SirFancyBacon) [reply src/main/resources/rs117/hd/scene/model_overrides.json:2754]: Oh good catch! 
It was meant to be metallic_1_light_highgloss

Metallic_2 was just metallic_1 lightened so I renamed it.

**Comment by SirFancyBacon** [comment]
Should have fixed the issues and brought inline with master

**Pull request merged**